images/virtualization-artifact/pkg/controller/cvi/cvi_reconciler.go

```diff
@@ -66,18 +69,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{}, nil
 	}
 
-	var requeue bool
+	log.Info("Start cvi reconciliation")
```

Comment:
Do we always want to know about the beginning of the reconciliation? Seems redundant for 'INFO'